### PR TITLE
Move billing section from KB site to docs

### PR DESCRIPTION
**Pull request by ruthnaebeck**
### What does this PR do?
- Move billing section from KB site to docs

### Motivation
KB site migration

### Preview link
https://docs-staging.datadoghq.com/ruth/billing/account_management/billing/

### CODE REVIEW

File: content/account_management/billing/log_management.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+## Tracking log events
+
+There are several places where you can see the number of log events you have sent to Datadog.
+
+1. On the [Usage page][1], there is a column which shows the hourly number of indexed log events:
+
+    {{< img src="account_management/billing/log-events01.png" alt="Log Events" responsive="true">}}
+
+2. On the [Pipeline page][2], double-click on an index to see the number of log events that were indexed in the past couple days.
```
Comment (l0k0ms) [inline content/account_management/billing/log_management.md]:
```suggestion
2. In the [Pipeline page][2], double-click on an index to see the number of log events that were indexed in the past couple of days.
```

File: content/account_management/billing/log_management.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+At the end of the month, we compute the total number of log events that have been indexed:
+
+- If you are below commitment, your bill stays the same.
+- If you over-consume, we subtract the committed amount and the **overages** are charged with a 50% premium.
+
+## Tracking log events
+
+There are several places where you can see the number of log events you have sent to Datadog.
+
+1. On the [Usage page][1], there is a column which shows the hourly number of indexed log events:
```
Comment (l0k0ms) [inline content/account_management/billing/log_management.md]:
```suggestion
1. In the [Usage page][1], there is a column which shows the hourly number of indexed log events:
```

File: content/account_management/billing/usage_details.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+kind: faq
+---
+
+<div class="alert alert-warning">
+This feature is only available to paying customers.
+</div>
+
+## Overview
+
+Admin users can access the [Usage][1] page by hovering over their username at the bottom left, then navigate to:
```
Comment (l0k0ms) [inline content/account_management/billing/usage_details.md]:
```suggestion
Administrators users can access the [Usage][1] page by hovering over their username at the bottom left, then navigate to:  
```

File: content/account_management/billing/log_management.md (hunk 2 above)
Comment (ruthnaebeck) [reply content/account_management/billing/log_management.md]: It's better to use `On` here.

File: content/account_management/billing/log_management.md (hunk 1 above)
Comment (ruthnaebeck) [reply content/account_management/billing/log_management.md]: It's better to use `On` here.

**Comment by ruthnaebeck** [comment]
@wissallahjouji has given approval to merge this.

**Pull request merged**